src/peer_digest.cc

```diff
@@ -65,19 +66,10 @@ static const time_t GlobDigestReqMinGap = 1 * 60;   /* seconds */
 
 static time_t pd_last_req_time = 0; /* last call to Check */
 
-PeerDigest::PeerDigest(CachePeer * p)
+PeerDigest::PeerDigest(CachePeer *p) :
```

Comment:
Two minor polishing touches since we have to modify this line:

```suggestion
PeerDigest::PeerDigest(CachePeer * const p):
```

I know that Squid is not consistent regarding that colon placement, but I consider the suggested variant more logical/natural than the "lonely colon" one in this context because this is how we write lists in English (and many other languages): a, b, c. And this is nothing but a list.